### PR TITLE
Bring words into main branch

### PR DESCRIPTION
**Pull request by chanson02**
We will need to figure out how to get this to run on windows. Requires the jansson and curl libraries.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
 main
+*.exe
+.vscode
+SDL2.dll
+src/
```
Comment (chanson02) [inline .gitignore:5]: @keegangunkel before you pull from this branch, copy your SDL2.dll and src to another directory. Once you do a pull paste them back in.

**Comment by chanson02** [comment]
Continuing to work on an API that will pretty much fully abstract the word bank into one function.

**Comment by chanson02** [comment]
This should be ready for main

**Comment by keegangunkel** [comment]
Looks great, nice stuff cooper

**Review by keegangunkel — APPROVED** [review]
Looks great!

**Pull request merged**